src/vaibhav/Mall.java

```diff
@@ -0,0 +1,54 @@
+package vaibhav;
+
+public class Mall {
+	
+	String customerName;
+	int items;
+	String paymentMethod;
+	boolean needBill;
+	boolean needCarryBag;
+	String section;
+	
+	void itemAddedToCart(int items) {
+		this.items = items;
+	}
+	void customerName(String customerName){
+		this.customerName = customerName;
+	}
+	void paymentDone(String paymentMethod) {
```

Comment:
here method name should be paymentDoneUsing() as per the requirement, no logical changes, just FYI code based on requirement.